README.md

```diff
@@ -427,6 +439,43 @@ Check out the implementations that the community has already produced [here](htt
 
 --------------------------------------------------------
 
+<a name="promises"></a>
+Promise Support
+---------------
+
+Each function taking a callback also can be used as a promise, if the callback is omitted. This applies for:
+
+- `db.get(key[, options])`
+- `db.put(key, value[, options])`
+- `db.del(key[, options])`
+- `db.batch(ops[, options])`
+- `db.batch().write()`
+
+The only exception is the `levelup` constructor itself, which if no callback is passed will lazily open the database backend in the background.
```

Comment:
Nitpick (and a pre-existing problem): throughout the readme we're mixing "database" /  "database store" / "store" / "database backend" / "LevelDB store".

Reply:
I think we should spend some time on the readme in general. It seems to me that it has too much information as well as outdated

Reply:
I agree! This should live in its own Issue/PR though

Reply:
See #456.